tests/integration/conftest.py

```diff
@@ -2,9 +2,10 @@
 
 import pandas as pd
 import pytest
+from _pytest.legacypath import TempdirFactory
```

Comment:
not related to this PR, but this is suggestive that we should switch to using `tmp_path`